packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -16,6 +19,22 @@ describe('Sayt Driver Plugin', () => {
       autocomplete: (query, config, callback) => null,
     };
     driver = new SaytDriverPlugin();
+    validConfig = {
+      config: {
+        collection: 'backup'
+      }
+    };
+    fetchEvent = {
+      detail: {
+        query: 'shirt',
```

Comment:
This can just be a variable on its own since you are injecting it into `saytDataPayload` below. Then you can reference it through the payload.